### ISSUE TITLE
tour: [REPLACE WITH SHORT DESCRIPTION]

### ISSUE THREAD

**Issue by Acatdef** [golang/tour#53]
Context: https://tour.golang.org/welcome/2
In other language's this page,needs a link to English tour,it would make man find English pages easy.
Change the title above to describe your issue and add your feedback here, including code if necessary

**Comment by alebaffa** [comment]
Yeah, I think the curator of the various tours should take care of this.

**Comment by kytrinyx** [comment]
Having looked around a bit, I think that this could be solved by making an update to this page: https://github.com/golang/tour/blob/master/TRANSLATE, and making sure that the commit message includes information that would impel translators to make the change when they see the change appear on https://groups.google.com/group/go-tour-commits
